### ISSUE TITLE
Welcome screen and login/create user components could be spruced up

### ISSUE THREAD

**Issue by isidorosp** [rotki/rotki#971]
## Abstract

The welcome and login screen is one of the first things users see when they open the application. Let's add some interesting elements and clean up the UI a bit to make rotki more inviting!

I have started doing some work on this and will share some examples so that we can discuss and vote in the coming days.

**Comment by isidorosp** [comment]
Some proposals: https://www.figma.com/file/WpbDRNT0Ra8RcPAStBnNUK/colorful-rotki?node-id=0%3A1

**Comment by isidorosp** [comment]
What do you guys think of these two options for the Login screen? Option B substitutes the "Create New Account" button with a more subtle link. The thought process is that most users would rarely be clicking a "Create New Account" and therefore it should be the lesser prominent element of the two.

Option A: 2 buttons on Login screen
![image](https://user-images.githubusercontent.com/27592/80925950-298cf000-8d94-11ea-8765-41be4f5d1c76.png)

Option B: 1 button on Login screen
![image](https://user-images.githubusercontent.com/27592/80925939-1aa63d80-8d94-11ea-9ba6-9c2d718071a7.png)

**Comment by LefterisJP** [comment]
@isidorosp The wording may be confusing @isidorosp on the second option. It sounds really like a standard web application account and may make users think that the account is made in a server.

**Comment by LefterisJP** [comment]
We got some comments on the welcome screens via Twitter:

https://twitter.com/LefterisJP/status/1256873078547779584
https://twitter.com/LefterisJP/status/1256924294984216578

Most people seemed to prefer lowercase. They don't like the brown in the color scheme in general. And they were split mostly between red and teal.

**Comment by isidorosp** [comment]
> @isidorosp The wording may be confusing @isidorosp on the second option. It sounds really like a standard web application account and may make users think that the account is made in a server.

This is a good point. Perhaps we can remove the question to reduce that confusion. Do you like/agree with the idea of de-emphasizing the Create an Account option by changing it from a button to a plain link, or would you like to go back just the two buttons?

**Comment by isidorosp** [comment]
> We got some comments on the welcome screens via Twitter:
> 
> https://twitter.com/LefterisJP/status/1256873078547779584
> https://twitter.com/LefterisJP/status/1256924294984216578
> 
> Most people seemed to prefer lowercase. They don't like the brown in the color scheme in general. And they were split mostly between red and teal.

Based on the feedback I propose the following:
- Implement the single color block header with our brown for now (I will include a screenshot in the PR)
- Keep the lowercase 'r' in 'rotki'. I will implement it here for the Welcome Screen elements and we can open another Issue to replace it throughout the app.
- In parallel to this we open the "theme face-lift" as its own issue (separate to this & the dashboard issues which should be more about content/layout /functionality rather than colors). I think if we do this in a certain manner the user would be able to choose from an "accent" color to apply through-out the app (e.g. red / teal / yellow) while keeping the general theme consistent.
